core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala

```diff
@@ -476,16 +476,16 @@ private[spark] class BlockManagerInfo(
       val blockStatus: BlockStatus = _blocks.get(blockId)
       _blocks.remove(blockId)
       if (blockStatus.storageLevel.useMemory) {
-        logInfo("Removed %s on %s in memory (size: %s, free: %s)".format(
+        logDebug("Removed %s on %s in memory (size: %s, free: %s)".format(
```

Comment:
These ones also seem useful in some cases for reasoning about dropping blocks due to cache contention.